src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -175,6 +175,219 @@ Query.select(["name", "title"])
 ```
 {% /multicode %}
 
+### Select relationship data {% #relationship-select %}
+
+With [opt-in relationship loading](/docs/products/databases/relationships#performance-loading), you must explicitly select relationship data. This gives you fine-grained control over performance and payload size.
+
+#### Get documents without relationships
+By default, documents return only their own fields:
+
+{% multicode %}
+```client-web
+const doc = await databases.getDocument('<DATABASE_ID>', '<COLLECTION_ID>', '<DOCUMENT_ID>');
```

Comment:
lets split all code examples like this into multi-line so we don't have to scroll horizontally.

Reply:
Sure, done